Calculator/Calculator/Controller/ViewController.swift

```diff
@@ -7,12 +7,198 @@
+        allInputScrollView.scrollToBottom()
+    }
+    
+    @IBAction func tapEqual(_ sender: UIButton) {
+        do {
+            if operationReady {
+                tapOperator(sender)
+                resetInputOperatorLabel()
+                
+                var formula = try ExpressionParser.parse(from: formulaString)
+                let result = try formula.result()
+                
+                inputNumberLabel.text = makeNumberFormat(for: String(result))
+                operationReady = false
+                
+                resetFormulaString()
+            }
+        } catch let error as ExpressionParserError {
+            switch error {
+            case .invalidOperand:
+                print("invalid operand")
+            }
+        } catch let error as FormulaError {
+            switch error {
+            case .operationFailure:
+                print("operation failure")
+            case .divideByZero:
+                inputNumberLabel.text = "NaN"
+                print("divide by zero")
+            }
+        } catch {
+            print("somthing wrong")
+        }
+    }
+    
+    @IBAction func tapChangeSign(_ sender: UIButton) {
+        let hyphenMinus = "-"
+        
+        if let numberLabelText = inputNumberLabel.text,
+           numberLabelText != "0" {
+            if numberLabelText.hasPrefix(hyphenMinus) {
+                inputNumberLabel.text = String(numberLabelText.dropFirst())
+            } else {
+                inputNumberLabel.text = hyphenMinus + numberLabelText
+            }
+        }
+    }
+    
+    @IBAction func tapClearEntry(_ sender: UIButton) {
+        resetInputNumberLabel()
+    }
+    
+    @IBAction func tapAllClear(_ sender: UIButton) {
+        resetInputNumberLabel()
+        resetInputOperatorLabel()
+        resetAllInputStackView()
+        operationReady = true
+    }
+    
+    private func resetInputNumberLabel() {
+        inputNumberLabel.text = "0"
+    }
+    
+    private func resetInputOperatorLabel() {
+        inputOperatorLabel.text = ""
+    }
+    
+    private func resetAllInputStackView() {
+        allInputStackView.subviews.forEach {
+            $0.removeFromSuperview()
+        }
+    }
+    
+    private func resetFormulaString() {
+        formulaString = ""
+    }
+    
+    private func makeUILabel(_ text: String?) -> UILabel {
+        let label = UILabel()
+        
+        label.text = text
+        label.textColor = .white
+        label.font = label.font.withSize(20)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        
+        return label
+    }
+    
+    private func makeUIStackView() -> UIStackView {
+        let stackView = UIStackView()
+        
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .fill
+        stackView.spacing = 8
+        
+        return stackView
+    }
```

Comment:
```suggestion
    private let label: UILabel = {
        let label = UILabel()
        
        label.text = text
        label.textColor = .white
        label.font = label.font.withSize(20)
        label.translatesAutoresizingMaskIntoConstraints = false
        
        return label
    }()
    
    private let stackView: UIStackView = {
        let stackView = UIStackView()
        
        stackView.translatesAutoresizingMaskIntoConstraints = false
        stackView.axis = .horizontal
        stackView.alignment = .fill
        stackView.distribution = .fill
        stackView.spacing = 8
        
        return stackView
    }()
```

`View`의 요소들을 메서드로 구성해주기 보다는 `프로퍼티`가 적절할 거 같습니다.